bodo/io/iceberg/catalog/dir.py

```diff
@@ -0,0 +1,254 @@
+import pyarrow as pa
```

Comment:
TODO: Add header.

Basically a lightweight copy of the HadoopCatalog from Java. We only support whats needed for basic testing